.CI/build_all

```diff
@@ -25,4 +25,4 @@ git ls-tree --name-only master -- . | xargs -I {} sh -c "rm -rf {} && echo Remov
 popd > /dev/null
 
 # We just want to build all of the recipes.
-conda-build-all ./recipes --matrix-condition "numpy >=1.11" "python >=2.7,<3|>=3.5"
+conda-build-all ./recipes --matrix-condition "numpy >=1.11" "python >=2.7,<3|>=3.5" "r-base >=3.3.2"
```

Comment:
> This would mean that we would need to adjust this to point to the latest version with any new R release.

Yes. Different from python build strings, the r-base build strings seems to be up to micro. For example:
```
conda search -c r r-oce
r-oce                        0.9_18                 r3.3.1_0  r               
                          .  0.9_20                 r3.3.2_0  r 
```

> Wasn't there an option in conda-build-all to only use the last *n* versions that satisfy the constraints?

`conda-build-all` will do *n* version. I am worried about that when I mentioned the CI strain above.